projects/core/public_api.ts

```diff
@@ -30,8 +33,4 @@ export * from './src/user/index';
 export * from './src/util/index';
 export * from './src/window/index';
 
-/** AUGMENTABLE_TYPES_START */
```

Comment:
Would you mind bringing back the grouping of exports under the magic comment?

The order in this file is crucial for _augmentable types_ feature to work (we run special post-builder for the types here under the magic comment) (see https://github.com/SAP/spartacus/issues/7940). AFAIK it's used only in core `public_api.ts` for now.

Btw. Thanks! You helped me notice how fragile is the mechanism and we will work on making it more bulletproof.

Reply:
Good hint, thanks..This is due the auto import organize which I should not run here